torch/csrc/lazy/ts_backend/ts_native_functions.cpp

```diff
@@ -433,113 +380,80 @@ at::Tensor & LazyNativeFunctions::normal_(at::Tensor & self, double mean, double
     // return self;
 };
 
-at::Tensor LazyNativeFunctions::permute(const at::Tensor& self,
-                                        at::IntArrayRef dims) {
+at::Tensor LazyNativeFunctions::_unsafe_view(const at::Tensor& self,
+                                     at::IntArrayRef size) {
   TORCH_LAZY_FN_COUNTER("lazy::");
-  auto self_tensor = torch::lazy::TryGetLtcTensor(self);
-  return torch::lazy::CreateAtenFromLtcTensor(torch::lazy::permute(
-      self_tensor, torch::lazy::ToI64Vector(dims)));
+  return LazyNativeFunctions::view_copy(self, size);
 }
 
-at::Tensor LazyNativeFunctions::select(const at::Tensor& self, int64_t dim,
-                                       int64_t index) {
-  TORCH_LAZY_FN_COUNTER("lazy::");
-  return torch::lazy::CreateAtenFromLtcTensor(
-      torch::lazy::select(torch::lazy::TryGetLtcTensor(self), dim, index));
-}
 
-at::Tensor LazyNativeFunctions::slice(const at::Tensor& self, int64_t dim,
-                                      c10::optional<int64_t> start,
-                                      c10::optional<int64_t> end,
-                                      int64_t step) {
-  int64_t start_val = start.has_value() ? start.value() : 0;
-  int64_t end_val = end.has_value() ? end.value() : INT64_MAX;
-  TORCH_LAZY_FN_COUNTER("lazy::");
-  return torch::lazy::CreateAtenFromLtcTensor(torch::lazy::slice(
-      torch::lazy::TryGetLtcTensor(self), dim, start_val, end_val, step));
+// This is needed by the torch.tensor constructor.
+// LazyTensor always opts into functionalization.
+// "lifting" a tensor for functionalization means wrapping it in a FunctionalTensorWrapper object.
+at::Tensor LazyNativeFunctions::lift(const at::Tensor& tensor) {
```

Comment:
Seeing as these implementations aren't specific to the TS backend and will likely be (copied and) used by the XLA backend and the MLIR backend too, would it be possible to move the implementations to somewhere common and export the symbols?